atd-toolbox/data_model/python/tests/tests.sql

```diff
@@ -0,0 +1,26 @@
+-- location A187DE10F4, in_austin_full_purpose true, council_district 9, engineering_area 2
+insert into public.crashes_cris (
+    crash_id, latitude, longitude, created_by, updated_by, cris_schema_version)
+    values (1, 30.2800238, -97.743370, 'cris', 'cris', '2023');
+insert into public.units_cris (unit_nbr, crash_id, cris_schema_version) values (1, 1, '2023');
+insert into public.people_cris (prsn_nbr, unit_nbr, crash_id, cris_schema_version) values (1, 1, 1, '2023');
+insert into public.charges_cris (prsn_nbr, unit_nbr, crash_id, charge, cris_schema_version) values (1, 1, 1, 'whatever', '2023');
+
+-- location null, in_austin_full_purpose false, council_district null
+update public.crashes_cris set active_school_zone_fl = TRUE, latitude = null where crash_id = 1;
+update public.units_cris set vin = 'abcdefg' where id = 1;
+update public.people_cris set drvr_city_name = 'austin' where id = 1;
```

Comment:
at first i thought the triggers on units/people werent working bc i was searching for where `crash_id = 1`. finally realized this was updating where `id = 1` :sweat_smile: all the diff id's still bother me but i think its the lesser of two evils. it does make sense to be keeping our own custom ids

Reply:
yeah 😵 we're going to be doing a lot of joining on people > units > crashes. i think we might just make ourselves a handy view that adds crash ID (and whatever else) to people records. in fact we'll probably need one for the socrata ETLs anyway. i also think we should find a way to subtly incorporate the person and unit IDs into the VZE so that it can be super easy to debug VZE issues 🤔

Reply:
ah, yes—these tests were designed to be run before the CRIS import increments all the sequences 😑 
i will try to re-write these in a way that isn't so fragile 🤔